docker-compose.yml

```diff
@@ -39,7 +39,7 @@ services:
       ALERTS_PROVIDER_ACCOUNT: ${ALERTS_PROVIDER_ACCOUNT-example_account}
       ALERTS_PROVIDER_PROJECT: ${ALERTS_PROVIDER_PROJECT-example_project}
       EMAIL_API_APPLICATION_CODE: ${EMAIL_API_APPLICATION_CODE-example_application_code}
-      EMAIL_API_FROM_EMAIL: ${EMAIL_API_FROM_EMAIL-example_from_email}
+      EMAIL_API_FROM_EMAIL: ${EMAIL_API_FROM_EMAIL-ch@nge.me}
```

Comment:
It's probably better to use `example.com` – https://www.iana.org/help/example-domains

Reply:
Thanks, I didn't know these domains are actually maintained by IANA 👍 Changed in [02d9ece](https://github.com/safe-global/safe-client-gateway/pull/942/commits/02d9ece0d4d8977a922c423eaccbe2d6a95e49d8)